src/graphics/draw/NodeListRenderer.cpp

```diff
@@ -139,7 +129,7 @@ void retrieveAndSortNodes(std::vector<NodeEntry> &nodeList)
         bool aFav = a.node->is_favorite;
         bool bFav = b.node->is_favorite;
         if (aFav != bFav)
```

Comment:
so if one node is a favourite (not neither, not both), then return the favourite status of node a?